webpack.common.config.ts

```diff
@@ -211,6 +213,17 @@ module.exports = {
       minify: htmlMinifyConfig,
       inject: false
     }),
+    new HtmlWebpackPlugin({
+      chunks: ['blog_dashboard'],
+      filename: 'blog-dashboard-page.mainpage.html',
+      hybrid: true,
```

Comment:
We will be using schema based editor which is still using angularjs. Thus `hybrid: true`